demo-config.ini

```diff
@@ -0,0 +1,22 @@
+[api]
+user     = donald.duck@ducktales.com
+password = superSECURE123
+
+[mail]
+from = sender@ducktales.com
+to = receiver@ducktales.com
+
+[mail.ducktales.com]
+snapshotStartDay = "friday"
+snapshotInterval = 21

```

Comment:
As I did not directly understand this value **snapshotInterval** , I would like have a commentation.
commentation in .ini-files with `; COMMENT`